src/AppRoutes.tsx

```diff
@@ -2,9 +2,10 @@ import React, { Suspense } from 'react';
 import { Route, Switch, withRouter } from 'react-router-dom';
 
 import { Layout, LayoutConfig } from '@features/Layout';
-import { Home, PageNotFound, ScreenLockProvider, DrawerProvider } from '@features';
-import { ScrollToTop, useScreenSize } from '@utils';
+import { Home, PageNotFound, ScreenLockProvider, DrawerProvider, MigrateLS } from '@features';
+import { ScrollToTop, useScreenSize, withContext } from '@utils';
 import { useFeatureFlags } from '@services';
+import { StoreContext, SettingsContext } from '@services/Store';
 import { ROUTE_PATHS } from '@config/routePaths';
 import {
   PageVisitsAnalytics,
```

Comment:
Import order

Reply:
I made a few changes.
IMHO: `@services` and `@services/Store` belong in the same import category.

Reply:
I made some minor changes.
IMHO `@services/Store` should be in the same import block as `@services`.